shims/emr700/driver/pom.xml

```diff
@@ -17,6 +17,8 @@
     <org.apache.hive.version>3.1.3</org.apache.hive.version>
     <org.apache.oozie.version>5.2.1</org.apache.oozie.version>
     <org.apache.hbase.version>2.6.1</org.apache.hbase.version>
+    <!-- settting v1.14.4 so shaded jackson-databind inside parquet-hadoop-bundle's  version approximately matches jackson-databind in driver.jar -->
+    <parquet.version>1.14.4</parquet.version>
```

Comment:
emr700 shim doesn't have a parquet-pid dependency, not sure why. main goal is to address parquet-hadoop-bundle